src/Common.Dotnet.CsWinRT.props

```diff
@@ -16,7 +16,7 @@
     <WarningLevel>4</WarningLevel>
     <NoWarn></NoWarn>
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <WarningsNotAsErrors>CA1720;CA1859;CA2263;CA2022;MVVMTK0045;MVVMTK0049</WarningsNotAsErrors>
+    <WarningsNotAsErrors>CA1824;CA1416;CA1720;CA1859;CA2263;CA2022;MVVMTK0045;MVVMTK0049</WarningsNotAsErrors>
```

Comment:
We have to add these two because the versioning project enables more warnings (I think?). These warnings are about using Windows APIs on "cross-platform" .NET projects. I think it is confused.